dbenv

```diff
@@ -45,20 +45,17 @@ get_port() {
 		return
 	fi
 
-	local cmd
-
-	cmd+="import socket;"
-	cmd+="s = socket.socket(socket.AF_INET, socket.SOCK_STREAM);"
-	cmd+="s.bind(('127.0.0.1', 0));"
-	cmd+="print(s.getsockname()[1]);"
-
-	if [ -n "$(which python)" ]; then
-		echo ${cmd} | python
-	elif [ -n "$(which python3)" ]; then
-		echo ${cmd} | python3
-	else
-		echo ${cmd} | python2
-	fi
+	local py
```

Comment:
`local py=python` in order to have a fallback and clearly complain if python isn't installed.